src/ontology/edits/oeo-social.omn

```diff
@@ -872,26 +885,35 @@ Individual: OEO_00010038
 
     Annotations: 
         OEO_00010037 "CRF 1",
-        <http://purl.obolibrary.org/obo/IAO_0000115> "The IPCC sector energy is an energy sector defined by the 2006 IPCC Guidelines for National Greenhouse Gas Inventories: All GHG emissions arising from combustion and fugitive releases of fuels. Emissions from the non-energy uses of fuels are generally not included here, but reported under Industrial Processes and Product Use.",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "The CRF sector (IPCC 2006) energy is a sector defined by the 2006 IPCC Guidelines for National Greenhouse Gas Inventories:
+All GHG emissions arising from combustion and fugitive releases of fuels. Emissions from the non-energy uses of fuels are generally not included here, but reported under Industrial Processes and Product Use.",
+        <http://purl.obolibrary.org/obo/IAO_0000119> "2006 IPCC Guidelines for National Greenhouse Gas Inventories, Volume 2, Chapter 2.2, https://www.ipcc-nggip.iges.or.jp/public/2006gl/",
         <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/461
 pull request: https://github.com/OpenEnergyPlatform/ontology/pull/523",
-        rdfs:label "CRF sector: energy"
+        rdfs:label "CRF sector (IPCC 2006): energy"
     
     Types: 
         OEO_00000367
     
     Facts:  
-     OEO_00000504  OEO_00000243
+     OEO_00000504  OEO_00000243,
```

Comment:
Why are the IPCC 2006-sectors defined by IPCC 1996 instead of IPCC 2006? That seems a bit unintuitive to me.

Reply:
It's a bug, well spotted. I fix it.